src/channel.d.ts

```diff
@@ -61,7 +61,7 @@ export type ChannelName =
  * An object literal of channel definitions. This is also used to represent
  * materialized channel states after mark initialization.
  */
-export type Channels = {[key in ChannelName]?: Channel};
+export type Channels = Record<string, Channel>;
```

Comment:
I’ve changed this here to allow specifying custom channels for tooltips, but we might want to make this change in more places… and we might even want to change the ChannelName type to include string so that it’s always allowed, but use the never hack to still let VS Code autocomplete known channel names. Although, that makes it less likely we will detect typos.